beets/ui/__init__.py

```diff
@@ -367,6 +367,31 @@ def input_yn(prompt, require=False):
     return sel == 'y'
 
 
+def input_select_items(prompt, items, rep):
+    """Prompts the user to use all, none or some of the items
+    Will return the list of items the user selected
+    prompt: prompt to use for all and for selective choice
+    items: full list of items
+    rep: function which represents an item to the user
+         is called with the item as argument
+         function is responsive for newline at input
+    """
+    out_items = []
+    choice = input_options(
+        ('y', 'n', 's'), False,
+        '%s (Yes/No/Select items)?' % prompt)
```

Comment:
Again on the detail-obsession topic: most of the other confirmation prompts in beets seem to have the form "Question? (A/b/c)" rather than "Question (A/b/c)?", so we should probably try for consistency with the placement of the question mark.

Also, we usually format this kind of prompt using just the initial letters, one of which is capitalized to indicate that it's the default. Of course, that isn't as easy to read when the options aren't just Y and N, which are self-evident. It's possible that we need some creativity here—how can we make it clear that there's a default, that you can just type one letter, and yet also that there's a third option?